### PR TITLE
update installation note with Mac M4

### PR DESCRIPTION
**Pull request by LinglongQian**

### CODE REVIEW

**Comment by LinglongQian** [comment]
update installation note with Mac M4

**Review by dakinggg — APPROVED** [review]
Thank you!

File: README.md [hunk 1]
```diff
@@ -30,13 +30,13 @@ Additionally, scispacy uses modern features of Python and as such is only availa
...
 Over the years, installing nmslib has becomes quite difficult. There are a number of GitHub issues on scispaCy and the nmslib repo itself about this. This matrix is an attempt to help users install nmslib in whatever environment they have. I don't have access to every type of environment, so if you are able to test something out, please open an issue or pull request!
 
-|               | Windows 11 | Windows Subsystem for Linux | Mac M1  | Mac M2  | Mac M3  | Intel Mac |
-|---------------|------------|----------------------------|---------|---------|---------|-----------|
-| Python 3.8    | ✅         | ✅                         | 💻      | ❓      | ❓      | ❓         |
-| Python 3.9    | ❌🐍       | ✅                         | 💻      | ❌🔨     | ❓      | ❓         |
-| Python 3.10   | ❌🐍       | ✅                         | ❓      | ❓      | ❓      | ✅         |
-| Python 3.11   | ❌🐍       | ❌🐍                       | ❓      | ❓      | ❓      | ❌         |
-| Python 3.12   | ❌🐍       | ❌🐍🧠                     | ❓      | ❓      | ❓      | ❓         |
+|               | Windows 11 | Windows Subsystem for Linux | Mac M1  | Mac M2  | Mac M3  | Mac M4  | Intel Mac |
```
Comment (dakinggg) [inline README.md:33]:
```suggestion
|               | Windows 11 | Windows Subsystem for Linux | Mac M1  | Mac M2  | Mac M3  | Mac M  | Intel Mac |
```
just making a change to retrigger CI, will put back immediately after

File: README.md [hunk 2]
```diff
@@ -30,13 +30,13 @@ Additionally, scispacy uses modern features of Python and as such is only availa
...
 Over the years, installing nmslib has becomes quite difficult. There are a number of GitHub issues on scispaCy and the nmslib repo itself about this. This matrix is an attempt to help users install nmslib in whatever environment they have. I don't have access to every type of environment, so if you are able to test something out, please open an issue or pull request!
 
-|               | Windows 11 | Windows Subsystem for Linux | Mac M1  | Mac M2  | Mac M3  | Intel Mac |
-|---------------|------------|----------------------------|---------|---------|---------|-----------|
-| Python 3.8    | ✅         | ✅                         | 💻      | ❓      | ❓      | ❓         |
-| Python 3.9    | ❌🐍       | ✅                         | 💻      | ❌🔨     | ❓      | ❓         |
-| Python 3.10   | ❌🐍       | ✅                         | ❓      | ❓      | ❓      | ✅         |
-| Python 3.11   | ❌🐍       | ❌🐍                       | ❓      | ❓      | ❓      | ❌         |
-| Python 3.12   | ❌🐍       | ❌🐍🧠                     | ❓      | ❓      | ❓      | ❓         |
+|               | Windows 11 | Windows Subsystem for Linux | Mac M1  | Mac M2  | Mac M3  | Mac M  | Intel Mac |
```
Comment (dakinggg) [inline README.md:33]:
```suggestion
|               | Windows 11 | Windows Subsystem for Linux | Mac M1  | Mac M2  | Mac M3  | Mac M4  | Intel Mac |
```

**Pull request merged**